worlds/sc2wol/Options.py

```diff
@@ -130,19 +132,10 @@ class ExcludedMissions(OptionSet):
 }
 
 
-def get_option_value(multiworld: MultiWorld, player: int, name: str) -> int:
-    option = getattr(multiworld, name, None)
+def get_option_value(multiworld: MultiWorld, player: int, name: str) -> int or FrozenSet:
```

Comment:
`int or FrozenSet` is just True.
It would be `int | FrozenSet` starting in Python 3.9, until then it's `typing.Union[int, FrozenSet]`

Reply:
Fixed.